saq/queue/postgres.py

```diff
@@ -128,14 +132,58 @@ async def init_db(self) -> None:
                 {"key1": self.saq_lock_keyspace},
             )
             result = await cursor.fetchone()
-
             if result and not result[0]:
                 return
 
-            for statement in DDL_STATEMENTS:
-                await cursor.execute(
-                    SQL(statement).format(jobs_table=self.jobs_table, stats_table=self.stats_table)
-                )
+            await cursor.execute(
+                SQL(
+                    dedent("""
+            CREATE TABLE IF NOT EXISTS {versions_table} (
+                version INT 
+            );
+                        """)
+                ).format(versions_table=self.versions_table)
+            )
+
+            migrations = get_migrations(
+                jobs_table=self.jobs_table,
+                stats_table=self.stats_table,
+            )
+            target_version = len(migrations)
```

Comment:
let's get the last one, and infer the version from the file name

Reply:
can we make this either a hash value or something less error prone? or perhaps like manual tagging of versions

Reply:
I don't think we can do hash cause we need to have total order

Reply:
he's gonna change it to be manual indices